client/src/components/Pages/Help/index.js

```diff
@@ -26,11 +28,51 @@ const HelpPage = () => {
     ))
   );
 
+  const renderLinks = () => (
+    <ul className="shortcut-links-wrapper">
+      <li>
+        <a
```

Comment:
I am using `<a>` here instead of `<Link>` because the styleguidist and the storybook are served through the Express server, not the React router. And the 404 page is the part of the React Router switch, so it should be a `<Link>` to work correctly.